data/layout/telescope_positions-South-4LST.ecsv

```diff
@@ -51,8 +46,8 @@
 #       unit: *id001
 #       value: 3.0
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-LST-01 57.5 57.5 57.5 -24.68291005156705 -70.31691313262846 366763.96140825993 7269522.935123434 2131.0
-LST-02 -57.5 -57.5 -57.5 -24.683948255723298 -70.31691313733373 366765.0646964067 7269407.961204613 2131.0
-LST-03 57.5 57.5 57.5 -24.68291005156705 -70.31577685466934 366878.93530575535 7269524.038389416 2131.0
-LST-04 -57.5 -57.5 -57.5 -24.683948255723298 -70.31577684996405 366880.0385930623 7269409.064514085 2131.0
+telescope_name      pos_x      pos_y      pos_z
+LST-01               57.5       57.5      57.5
+LST-02              -57.5      -57.5      57.5
+LST-03               57.5       57.5      57.5
```

Comment:
This telescope is sitting on top of the first one. I think that's a typo and either the `x` or `y` coordinates should be negative.